conf/routes

```diff
@@ -15,8 +15,8 @@ POST /monthly-contributions/create                  controllers.MonthlyContribut
 GET /oneoff-contributions/thankyou                  controllers.Application.reactTemplate(title="Support the Guardian | Thank You", id="oneoff-contributions-thankyou-page", js="oneoffContributionsThankyouPage.js")
 GET /uk/contribute              controllers.Application.reactTemplate(title="Support the Guardian | Make a Contribution", id="contributions-landing-page-uk", js="contributionsLandingPageUK.js")
 
-GET /oneoff-contributions                         controllers.OneOffContributions.displayForm
-GET /oneoff-contributions/test-user               controllers.OneOffContributions.displayFormTestUser
+GET /oneoff-contributions                         controllers.OneOffContributions.displayForm(paypal : Option[Boolean])
```

Comment:
Standard style is for there to be no space between the variable and colon:
`paypal: Option[Boolean]` (also line 19)